src/com/palmergames/bukkit/config/migration/Migration.java

```diff
@@ -0,0 +1,21 @@
+package com.palmergames.bukkit.config.migration;
+
+import com.palmergames.bukkit.util.Version;
+
+import java.util.List;
+
+/**
+ * Represents a collection of changes.
+ */
+class Migration {
```

Comment:
I think this class is small enough just to be a class inside `ConfigMigrator.java`